### ISSUE TITLE
Variants retrieved by /variants/{id} have empty `calls` field

### ISSUE THREAD

**Issue by hjellinek** [ga4gh/server#619]
Compliance test `VariantsGetByIdIT.checkVariantsGetResultsMatchSearchResults` does the following:

1. Call `POST /variants/search` to retrieve all `Variants` in a given start/end range on a given `VariantSet`.
1. For each `Variant` object in that list, call `GET /variants/{id}` with its ID
1. Test that the `Variant` from the search is equal to the one retrieved by ID

Expect: the objects are equal (meaning their fields contain the same information according to Java's `Object.equal`).

Actual: the objects returned by `GET /variants/{id}` have a `calls` field that contains the empty list, unlike the ones returned by `POST /variants/search`.

Specifically, here's a `Variant` returned by `/variants/search`:
```json
{"id": "YnJjYTE6MWtnUGhhc2UzOnJlZl9icmNhMTo1MDo5ZTY0ZDAyMzk3OTc0NzNjMjY5NjcxYjc3MzI4NTFjYw==",
"variantSetId": "YnJjYTE6MWtnUGhhc2Uz",
"names": ["rs8176320"],
"created": 1440623891000,
"updated": 1440623891000,
"referenceName": "ref_brca1",
"start": 50,
"end": 51,
"referenceBases": "C",
"alternateBases": ["T"],
"info": {"AA": ["C|||"], "EUR_AF": ["0.00989999994636"], "AMR_AF": ["0.00719999987632"], "AC": ["0"], "SAS_AF": ["0.00310000008903"], "EAS_AF": ["0.0"], "AF": ["0.00419329013675"], "NS": ["2504"], "AFR_AF": ["0.00230000005104"], "DP": ["19947"], "AN": ["6"], "VT": ["SNP"]},
"calls":
[{"callSetName": "HG00096", "callSetId": "YnJjYTE6MWtnUGhhc2UzOkhHMDAwOTY=", "genotype": [0, 0], "phaseset": "*", "genotypeLikelihood": [], "info": {}},
{"callSetName": "HG00099", "callSetId": "YnJjYTE6MWtnUGhhc2UzOkhHMDAwOTk=", "genotype": [0, 0], "phaseset": "*", "genotypeLikelihood": [], "info": {}},
{"callSetName": "HG00101", "callSetId": "YnJjYTE6MWtnUGhhc2UzOkhHMDAxMDE=", "genotype": [0, 0], "phaseset": "*", "genotypeLikelihood": [], "info": {}}]}
```
Here's the one returned by `GET /variants/{id}`:
```json
{"id": "YnJjYTE6MWtnUGhhc2UzOnJlZl9icmNhMTo1MDo5ZTY0ZDAyMzk3OTc0NzNjMjY5NjcxYjc3MzI4NTFjYw==",
"variantSetId": "YnJjYTE6MWtnUGhhc2Uz",
"names": ["rs8176320"],
"created": 1440623891000,
"updated": 1440623891000,
"referenceName": "ref_brca1",
"start": 50,
"end": 51,
"referenceBases": "C",
"alternateBases": ["T"],
"info": {"AA": ["C|||"], "EUR_AF": ["0.00989999994636"], "AMR_AF": ["0.00719999987632"], "AC": ["0"], "SAS_AF": ["0.00310000008903"], "EAS_AF": ["0.0"], "AF": ["0.00419329013675"], "NS": ["2504"], "AFR_AF": ["0.00230000005104"], "DP": ["19947"], "AN": ["6"], "VT": ["SNP"]},
"calls": []}
```

**Comment by jeromekelleher** [comment]
Excellent point @hjellinek, well caught.

**Comment by jeromekelleher** [comment]
Fixed in #624.
